test/posts.test.js

```diff
@@ -32,6 +33,7 @@ describe('test /posts endpoint', () => {
 
     expect(res.status).toEqual(200);
     expect(res.get('Content-type')).toContain('application/json');
+    expect(res.get('X-Total-Count')).toContain(createdItems.toString());
```

Comment:
Why `toContain` and not `toBe`?

Reply:
Oh, my bad. I copied and pasted the line before mine and changed the values (since it's almost the same line), and I forgot to change `toContain` to `toBe`. Thank you for catching that one @humphd.